snippets/terraform/deployments/create-or-update/main.tf

```diff
@@ -24,6 +24,7 @@ resource "azurerm_nginx_deployment" "example" {
   resource_group_name      = module.prerequisites.resource_group_name
   sku                      = var.sku
   location                 = var.location
+  capacity                 = 20
```

Comment:
change this to 50 maybe.

Reply:
Yes - please update it to be something other than the default value - preferably something higher than 20. I am okay with 50.

Reply:
done 👍